sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -37,7 +37,7 @@ public static HttpPipeline Build(ClientOptions options, params HttpPipelinePolic
             ClientOptions options,
             HttpPipelinePolicy[] perCallPolicies,
             HttpPipelinePolicy[] perRetryPolicies,
-            ResponseClassifier responseClassifier)
+            ResponseClassifier? responseClassifier)
```

Comment:
This reflects the actual behavior we have. We never checked for nulls and used the default classifier.